.github/workflows/analytics_migrations.yml

```diff
@@ -0,0 +1,33 @@
+name: Check and Run Analytics Migrations
```

Comment:
Does this workflow actually work? It seems like the DB is inside the private VPC, unless I'm missing something.

Reply:
You are correct. I spent some time coming up with a way around this, but have since backed it out of this PR, to avoid more unnecessary complexity and delays. I will address automatically running migrations in a follow up PR.